tests/Fixtures/Integration/priority/braces,array_indentation.test

```diff
@@ -1,5 +1,5 @@
 --TEST--
-Integration of fixers: array_indentation,braces.
```

Comment:
:-1:
the idea was that order of rules in file name should be alphabetical, to ease searching

Reply:
Really? I would never have guessed, and counting the number of tests having not alphabetical order - which is 11 - it looks like you are the only one knowing that 😉 

Anyway, what would that ease searching for? I have spotted this when working at https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5563 - I've added test file `single_space_after_construct,braces.test` and saw it's right after the `single_import_per_statement` ones, so I've added the entry in `tests/AutoReview/FixerFactoryTest.php` right after the `single_import_per_statement` ones, and got not nice error as there is an entry with `single_line_throw` fixer.

Reply:
i don't mind which way we would go, but lets make it explicit - either always alphabetical, either in order of execution.
if we allow for both, we can end up with 2 files

Reply:
The change in `tests/AutoReview/FixerFactoryTest.php` solves exactly that concern.